src/test/java/net/sf/jabref/logic/layout/LayoutTest.java

```diff
@@ -86,15 +82,6 @@ public void testHTMLChar() throws IOException {
     }
 
     @Test
-    @Ignore
```

Comment:
In other words: That is the current behavior, yes. But should it really be the correct behavior? In my opinion we should only test correct behavior not as is behavior. As the correct behavior for example <br> is not working right now I cannot really test it.